api/tests/routes/pro/patch_venue_test.py

```diff
@@ -135,22 +135,52 @@ def test_edit_venue_withdrawal_details_with_applied_on_all_offers(
                 "name": "new name",
                 "withdrawalDetails": "Ceci est un texte de modalités de retrait",
                 "isWithdrawalAppliedOnAllOffers": True,
+                "shouldSendMail": True,
             },
             venue,
         )
-
         response = auth_request.patch("/venues/%s" % humanize(venue.id), json=venue_data)
 
         assert response.status_code == 200
         assert venue.withdrawalDetails == "Ceci est un texte de modalités de retrait"
 
         mocked_update_all_venue_offers_withdrawal_details_job.assert_called_once_with(
-            venue, "Ceci est un texte de modalités de retrait"
+            venue, "Ceci est un texte de modalités de retrait", send_email_notif=True
         )
 
         assert len(external_testing.sendinblue_requests) == 1
         assert len(external_testing.zendesk_sell_requests) == 1
 
+    @patch("pcapi.routes.pro.venues.update_all_venue_offers_withdrawal_details_job.delay")
```

Comment:
<!-- Issues represent user-facing problems. If possible, it's great to follow this kind of comment with a suggestion.  -->
**🔨 issue:** ‏Pareil, ici on a pas besoin du mock
[Le décorateur](https://github.com/pass-culture/pass-culture-main/blob/7acc266067f9b7de66b34dc827d0da1651b395c5/api/src/pcapi/tasks/decorator.py#L36) `task` s'occupe de faire le bon call en état de test

Reply:
yep mais là ce que je cherche à faire c'est vérifier que le job est appelé avec les bon arguments. il me faut donc un mock pour vérifier la valeur passée au job pour `send_email_notification`